libraries/botbuilder-ai/src/qnaMakerDialog.ts

```diff
@@ -113,6 +114,24 @@ export interface QnAMakerDialogConfiguration extends DialogConfiguration {
     rankerType?: RankerTypes | string | Expression | EnumExpression<RankerTypes>;
 }
 
+/**
+ * Returns an activity with active learning suggestions.
+ *
+ * Important: The activity returned should relay the noMatchesText as an option to the end user.
+ *
+ * @param suggestionsList List of suggestions.
+ * @param noMatchesText If this text is received by the bot during a prompt.
+ */
+export type QnASuggestionsActivityFactory = (suggestionsList: string[], noMatchesText: string) => Partial<Activity>;
+
+const isSuggestionsFactory: Test<QnASuggestionsActivityFactory> = (val): val is QnASuggestionsActivityFactory => {
+    return tests.isFunc(val);
+};
+
+const isSuggestionsActivity: Test<Activity> = (val): val is Activity => {
```

Comment:
Is this supposed to be `Partial<Activity>`? I think there is already an activity type assertion and a partial activity type assertion in schema or core.